### ISSUE TITLE
noisy input in Chime-3 dataset

### ISSUE THREAD

**Issue by HyperbolicTangent** [sekiguchi92/SoundSourceSeparation#8]
Hello,

Recently I have read your awesome paper "Bayesian Multichannel Speech Enhancement with a Deep Speech Prior" and I want to use your score in the paper. However, when I calculate the average SDR value of Chime-3 development input data, the input SDR is very low which is about -4.5dB, far from 5.8dB reported in the paper. I used BSS_eval python implementation to calculate SDR. Did you use any pre-processing methods to normalize or align the input audio? Or 5.8dB is calculated just from the raw audio? 
I would appreciate it if you could give me some suggestions.

**Comment by sekiguchi92** [comment]
Hello.

I'm sorry for late reply, and thank you for having interest in my research.

I did not do any pre-processing for calculating the SDR of input.
What method did you use for evaluation? bss_eval_images or bss_eval_sources?
I used bss_eval_sources in my experiment.

If you used bss_eval_images, this may be the cause of the low SDR,
because in bss_eval_images, the scale of signal affects the result.
